R/utility.R

```diff
@@ -164,6 +174,16 @@ readSeqWareTSV <- function(path) {
     set(dt, j = "Run Name", value = factor(rep(path, nrow(dt))))
 
   }, warning = function(w) {
+    # This is a crutch to fix a bug in data.table. If fread causes a warning, it is caught, but the function is left in a failed state.
+    # The next valid call to fread will fail again, as the function has to reset itself. The next valid call will work fine.
+    # I am manually making a valid call to fread to reset it
+    # Issue is at https://github.com/Rdatatable/data.table/issues/2904
+    tryCatch({
```

Comment:
:scream: That's quite a bug.

Reply:
3 hours of my life right there. The good news is that my Issue was deemed a bug, so there was a purpose to my hair pulling.

Reply:
Thanks for the report.  10 _years_ of my life hair pulling on data.table now.